### PR TITLE
Add BASH completion

### PR DESCRIPTION
**Pull request by ericem**
I added support for BASH completion. It will try and match any desk files that you have installed, which makes switching desks really quick.

### CODE REVIEW

File: completions/bash/desk [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+_desk() {
+    PREFIX="${DESK_DIR:-$HOME/.desk}"
+    DESKS="${DESK_DESKS_DIR:-$PREFIX/desks}"
+
+    cur=${COMP_WORDS[COMP_CWORD]}
+    prev=${COMP_WORDS[COMP_CWORD-1]}
+
+    case ${COMP_CWORD} in
+        1)
+            COMPREPLY=($(compgen -W "edit go help init list ls version" ${cur}))
```
Comment (jamesob) [inline completions/bash/desk]: missing `.` here

File: completions/bash/desk [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            ;;
+        2)
+            case ${prev} in
+                edit|del|go)
```
Comment (jamesob) [inline completions/bash/desk]: `del`?

**Comment by jamesob** [comment]
This is great, thanks. Just a few minor items.

File: completions/bash/desk (hunk 2 above)
Comment (ericem) [inline completions/bash/desk]: Oops, I was working on delete support and this snuck in too early.

**Comment by jamesob** [comment]
Thanks, @ericem.

**Pull request merged**